mqtt/mqtt-bridge/src/controller/bridges.rs

```diff
@@ -39,7 +39,7 @@ pub(crate) struct Bridges {
 impl Bridges {
     pub(crate) async fn start_bridge<S>(&mut self, bridge: Bridge<S>, settings: &ConnectionSettings)
     where
-        S: StreamWakeableState + Send + 'static,
+        S: StreamWakeableState + Send + 'static + Sync,
```

Comment:
Even after format it didn't put lifetime last.. not sure if this is a style issue or not, but will change if it is